src/inlineSuggestions/positionExtracter.ts

```diff
@@ -0,0 +1,55 @@
+import { EOL } from "os";
```

Comment:
`positionExtracter` ?

Reply:
do you have a better name? it extracts the position from the change...